interfaces/builtin/basedeclaration.go

```diff
@@ -441,6 +441,11 @@ slots:
     deny-auto-connection: true
     deny-connection:
       on-classic: false
+  online-accounts-service:
+    allow-installation:
+      slot-snap-type:
+        - app
+    deny-connection: true
```

Comment:
This was mostly talking about how auto-connection is allowed (because it is omitted), but to make this clear, the base declaration is saying that:
- the slot implementation may only be supplied by app snaps
- slot implementations by default may not use this interface
- plugging snaps are auto-connected by default

*If* there is a slot implementation that is deemed safe and operates how the interface is designed (and safe for auto-connection), then a snap declaration can be issued to the slot implementation snap allowing it to use the interface.